contracts/libraries/icq-querier/src/contract.rs

```diff
@@ -0,0 +1,209 @@
+use std::collections::BTreeMap;
```

Comment:
I think we may want to put this behind a more generic KV interface because we'll likely be using a different tree when we move to risc